sdk/storage/storage-blob/tsconfig.json

```diff
@@ -1,26 +1,11 @@
 {
+  "extends": "../../../tsconfig.package",
   "compilerOptions": {
-    "alwaysStrict": true,
-    "noImplicitAny": true,
-    "preserveConstEnums": true,
-    "sourceMap": true,
-    "inlineSources": true,
-    "newLine": "LF",
     "target": "es5",
-    "moduleResolution": "node",
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "strict": true,
-    "module": "esNext",
-    "outDir": "./dist-esm",
-    "declaration": true,
-    "declarationMap": true,
-    "importHelpers": true,
     "declarationDir": "./typings/latest",
-    "lib": ["dom", "es5", "es6", "es7", "esnext"],
-    "esModuleInterop": true
+    "outDir": "./dist-esm",
+    "lib": ["dom", "es5", "es6", "es7", "esnext"]
```

Comment:
I'm not going to ask you to fix this here, but we really need to clean this up at some point... if they're relying on supporting IE11 they can't just import esnext from libs...

@jeremymeng can you file a bug on this?

Reply:
Opened an issue to track this - https://github.com/Azure/azure-sdk-for-js/issues/13669